app/queries/tax_rates_query.rb

```diff
@@ -6,6 +6,11 @@ def call(search_term:, page:, limit:, filters: {})
 
     tax_rates = base_scope.result
     tax_rates = tax_rates.where(id: filters[:ids]) if filters[:ids].present?
+
+    unless filters[:applied_by_default].nil?
```

Comment:
maybe using `if filters[:applied_by_default].present?` to be consistent with the previous line related to `ids`?